src/ngraph/op/topk.cpp

```diff
@@ -443,21 +443,197 @@ void op::v1::TopK::set_k(size_t k)
         op::Constant::create(element::i64, Shape{}, {k})->output(0));
 }
 
-namespace ngraph
+// v3 version starts
+constexpr NodeTypeInfo op::v3::TopK::type_info;
+
+op::v3::TopK::TopK(const Output<Node>& data,
```

Comment:
Since the TopK op already supported the output indices type I will instead update the specification and use  'index_element_type' as an attribute name. Then let's use inheritance to create a version3 op. This will significantly reduce the amount of code.

Reply:
The [specification](https://gitlab-icv.inn.intel.com/inference-engine/dldt/merge_requests/7010/) has just been updated.

Reply:
Done.